crates/paralegal-policy/src/flows_to.rs

```diff
@@ -47,54 +62,100 @@ impl CtrlFlowsTo {
             }
         }
 
+        fn iterate(
+            sources: &Arc<IndexedDomain<DataSource>>,
+            flows_to: &mut IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
+            sink_to_source: &IndexMatrix<CallSiteOrDataSinkIndex, DataSource>,
+        ) {
+            // Compute the transitive closure to a fixpoint.
```

Comment:
Since this is now a separate function, this comment should probably become a documentation comment on `iterate`

Reply:
Done